src/luascript.cpp

```diff
@@ -2553,10 +2557,56 @@ void LuaScriptInterface::registerFunctions()
 
 	registerMethod("Condition", "addDamage", LuaScriptInterface::luaConditionAddDamage);
 
+	// NewMonsterType
+	registerClass("NewMonsterType", "", LuaScriptInterface::luaNewMonsterTypeCreate);
+
 	// MonsterType
 	registerClass("MonsterType", "", LuaScriptInterface::luaMonsterTypeCreate);
```

Comment:
let's hear some more people what they think about it, I'm not against it at all but atm rather both are not the best way, I'd really like to have more feedback about that first :)